Sources/StreamChat/Models/Channel.swift

```diff
@@ -602,3 +646,164 @@ public extension ChatChannel {
         ownCapabilities.contains(.shareLocation)
     }
 }
+
+// MARK: - Payload -> Model Mapping
+
+extension ChannelPayload {
```

Comment:
nit: let's remove these in a separate file, this file becomes too big.

Reply:
Done